cassandra/io/eventletreactor.py

```diff
@@ -27,11 +26,24 @@
 from six.moves import xrange
 
 from cassandra.connection import Connection, ConnectionShutdown, Timer, TimerManager
+try:
+    from eventlet.green.OpenSSL import SSL
+    _PYOPENSSL = True
+except ImportError as no_pyopenssl_error:
+    _PYOPENSSL = False
 
 
 log = logging.getLogger(__name__)
 
 
+def _check_pyopenssl():
+    if not _PYOPENSSL:
+        raise ImportError(
+            "{}, pyOpenSSL must be installed to enable "
+            "SSL support with the Twisted event loop".format(str(no_pyopenssl_error))
+        )
```

Comment:
Woah, good catch. Looked into it and behavior is different between 2.7 and 3.x. 2.7 this would work fine, but in 3.x the exception goes out of scope after the except clause unless it's been reassigned to a new variable.